crates/nu-command/src/platform/clear.rs

```diff
@@ -24,28 +25,40 @@ impl Command for Clear {
         Signature::build("clear")
             .category(Category::Platform)
             .input_output_types(vec![(Type::Nothing, Type::Nothing)])
+            .switch("all", "Clear the terminal and its history", Some('a'))
```

Comment:
Just to reduce confusion with the shell history it may be worth clarifying that this is affecting the scrollback buffer.

```suggestion
            .switch("all", "Clear the terminal and its scroll-back history", Some('a'))
```